lib/tests/test_shape_utils.py

```diff
@@ -0,0 +1,142 @@
+import geopandas as gpd
```

Comment:
Why didn't understand GitHub that this file has been moved? Did you change much in here?

Reply:
Only two changes are: the location the util functions are found (now in `renewablepotentialslib`) and the lack of dependence on the default/schema config files for e.g. the possible NUTS years. Since this test is now in the lib, it needed completely unlinking from the config files, which I think is fine for the use they had (*only* to identify NUTS years)